projects/plugins/crm/tests/action-skip-test-php.sh

```diff
@@ -5,8 +5,8 @@ if php -r 'exit( version_compare( PHP_VERSION, "7.2.0", "<" ) ? 0 : 1 );'; then
 	exit 3
 fi
 
-# TODO Fix the bug and remove this
-if [[ "$WP_BRANCH" == 'trunk' ]]; then
-    echo "Temporarily disabling tests against WP trunk until we get around to updating our codeception DB image."
-    exit 3
-fi
+# Uncomment the below snippet to disable tests on WP trunk
+# if [[ "$WP_BRANCH" == 'trunk' ]]; then
+# 	echo "Codeception tests against WP trunk are temporarily disabled."
+# 	exit 3
+# fi
```

Comment:
Can we remove this as well? 😄

Reply:
I initially was planning to remove it (I typically don't like commented + unused code), but figured I'd leave it in our tooling as an example of a quick way to disable tests should something similar come up.